marshaling.go

```diff
@@ -349,11 +332,39 @@ func (value *Number) structScanConvert(ctx convertCtx) (reflect.Value, error) {
 }
 
 func (value *Worksheet) structScanConvert(ctx convertCtx) (reflect.Value, error) {
-	return ctx.cannotConvert("not supported yet")
+	if ctx.destType.Kind() != reflect.Struct {
+		return ctx.cannotConvert("dest must be a struct")
+	}
+	newVal := reflect.New(ctx.destType)
+	err := value.StructScan(newVal.Interface())
+	if err != nil {
+		return reflect.Value{}, err
+	}
+	return newVal.Elem(), nil
 }
 
 func (value *Slice) structScanConvert(ctx convertCtx) (reflect.Value, error) {
-	return ctx.cannotConvert("not supported yet")
+	if ctx.destType.Kind() != reflect.Slice && ctx.destType.Kind() != reflect.Array {
```

Comment:
Let's not support scanning into an array: they are used for low level things, like data structure work.

(Code will also be easier to read since there won't be two paths for dests.)